CMakeLists.txt

```
@@ -483,6 +483,8 @@ if (OPENMP_FOUND)
   add_definitions(-DHAS_OPENMP)
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
+  if(OpenMP_CXX_FOUND)
+    set(MLPACK_LIBRARIES ${MLPACK_LIBRARIES} ${OpenMP_CXX_LIBRARIES})
```

My bad; should have specified!
```suggestion
    set(MLPACK_LIBRARIES ${MLPACK_LIBRARIES} ${OpenMP_CXX_LIBRARIES})
  endif ()
```

---

Oops. I forgot about that

---

Nah, that's on me; I made an incomplete suggestion! Thanks for resolving so quickly.